src/Middleware/OutputCaching/src/Resources.resx

```diff
@@ -0,0 +1,123 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We currently have exactly one Resources.Designer.cs file in the repo, for a very special case. Why add src/Middleware/OutputCaching/src/Resources.Designer.cs❔ Everything that needs to be built will be automatically without it.

Reply:
> Why add src/Middleware/OutputCaching/src/Resources.Designer.cs

Thanks, didn't know.